WordPress/UITestsFoundation/Screens/Editor/EditorPostSettings.swift

```diff
@@ -77,12 +114,45 @@ public class EditorPostSettings: ScreenObject {
         return try EditorPostSettings()
     }
 
-    /// - Note: Returns `Void` because the return screen depends on which editor the user is in.
-    public func closePostSettings() {
+    @discardableResult
+    public func closePostSettings() throws -> BlockEditorScreen {
         navigateBack()
+
+        return try BlockEditorScreen()
     }
 
     public static func isLoaded() -> Bool {
         return (try? EditorPostSettings().isLoaded) ?? false
     }
+
+    @discardableResult
+    public func updatePublishDate() -> Self {
```

Comment:
A few notes here:

1. The function adjusts the time 2 hours forward, but neither this is reflected in the name, nor it's an argument. By looking at the test, it's not clear how exactly it's adjusted.
2. It adjusts the time, not the date 🙂
3. I did not test this IRL, but thinking of this case: if the time is `11 PM`, the adjusted time will be `01 AM`. However, the date will stay the same, which means the post will be scheduled for the past. As I think of alternative solutions, the only stable thing that comes to mind in tapping the `>` button in the calendar instead, and selecting any date from the next month.

Reply:
to address points 1 and 2, this was named from the name on the button and the action that the test is doing - updating the value using the "publish date" button:
<img width="498" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/17252150/a08365c3-1c4f-4540-b6ae-5e3cc6652d3c">

however, after considering your suggestion in point 3, i've decided to use your suggestion and updated it to select the first day of the next month instead. the name should be correct now (date instead of time). i also renamed it to `updatePublishDateToFutureDate` after finding out that the button also changes to `scheduled` when a past date is selected, changes in https://github.com/wordpress-mobile/WordPress-iOS/pull/21042/commits/e2740ab5f2374f94e4a6ed9e6977d646d10f49db

and i'll open an issue so we can confirm the correct behavior for scheduling posts in the past